barao/comida.tex

```diff
@@ -59,11 +59,10 @@ \subsubsection{Cardápio vegetariano}
 
 \subsubsection{RS na janta}
 
-O RS, inaugurado no fim de 2012, operou somente no período do almoço, até o
-fechamento desta edição do Manual. Porém, por conta de uma demanda do CACo,
-juntamente com outros centros acadêmicos e o DCE, o que levou a um
-abaixo-assinado e então a uma petição, foi autorizado o funcionamento do RS no
-período do jantar.
+O RS, inaugurado no fim de 2012, operou somente no período do almoço, até 2015.
+Porém, por conta de uma demanda do CACo, juntamente com outros centros
+acadêmicos e o DCE, o que levou a um abaixo-assinado e então a uma petição, foi
```

Comment:
Eu trocaria "foi autorizado o funcionamento do RS no período do jantar" por "agora o RS funciona também no período do jantar"

Ou até mesmo tiraria isso e só deixaria nas informações do RS que ele funciona de noite. (como isso é recente, pode ser melhor deixar pro próximo ano, not sure)